GliaWidgets/SecureConversations/SecureConversations.Coordinator.swift

```diff
@@ -11,9 +11,9 @@ extension SecureConversations {
             self.viewFactory = viewFactory
         }
 
-        func start() -> SecureConversations.WelcomeViewController {
+        func start() -> ViewController {
             let viewController = makeSecureConversationsWelcomeViewController()
-
+//            let viewController = makeSecureConversationsConfirmationViewController()
```

Comment:
could we remove this?

Reply:
I left it so that reviewers could test this if they want, but yes, better to remove it a8d51fce2e42f6b0de47b2f8889c70cb0ae29d94